core/types/calls.cc

```diff
@@ -1103,25 +1099,28 @@ TypePtr getMethodArguments(const GlobalState &gs, SymbolRef klass, NameRef name,
     return TupleType::build(gs, move(args));
 }
 
-TypePtr ClassType::getCallArguments(const GlobalState &gs, NameRef name) {
+TypePtr ClassType::getCallArguments(const GlobalState &gs, NameRef name) const {
     if (hasUntyped()) {
-        return Types::untyped(gs, untypedBlame());
+        return Types::untyped(gs, Symbols::noSymbol());
```

Comment:
Nice: it's OK to use `Symbols::noSymbol()` here because of the specialized implementation for `BlameUntyped` below.